extras/mockUI/libTC.cpp

```diff
@@ -51,11 +51,11 @@ float eeprom(uint8_t index) {
     case 0:
       return EEPROM_TC::instance()->getPh();
     case 1:
-      return EEPROM_TC::instance()->getTemp();
+      return EEPROM_TC::instance()->getThermalTarget();
     case 2:
       return EEPROM_TC::instance()->getTankID();
     case 3:
-      return EEPROM_TC::instance()->getCorrectedTemp();
+      return EEPROM_TC::instance()->getThermalCorrection();
     case 4:
```

Comment:
Nice. This is more than just "temp" cleanup: `getCorrectedX()` implies something different than `getXCorrection()`. You are also removing other ambiguities.